app/src/main/java/chat/rocket/android/push/interactors/DefaultPushInteractor.java

```diff
@@ -1,19 +0,0 @@
-package chat.rocket.android.push.interactors;
-
-import chat.rocket.android.helper.ServerPolicyHelper;
-import chat.rocket.android.model.ServerConfig;
-import chat.rocket.android.realm_helper.RealmStore;
-
-public class DefaultPushInteractor implements PushInteractor {
```

Comment:
I understood what you intended.

However I think the PushInteractor interface is not well-encapsulated.
I will try another improvement.
PushNotificationHandler already knows ServerConfig has an id with String type. If encapsulating the Realm detailed implementation,